addons/stock/models/stock_picking.py

```diff
@@ -191,6 +199,34 @@ def write(self, vals):
                     })
         return super(PickingType, self).write(vals)
 
+    @api.model
+    def _search_is_favorite(self, operator, value):
+        if operator not in ['=', '!='] or not isinstance(value, bool):
+            raise NotImplementedError(_('Operation not supported'))
+        return [('favorite_user_ids', 'in' if (operator == '=') == value else 'in', self.env.uid)]
```

Comment:
` 'in' if (operator == '=') == value else 'in'`

It looks like an error ;) Condition is either useless or one of the value is.

Reply:
Good point :sweat_smile: I wrote this a while ago, so I don't quite remember what was the idea here ... Could be also that, after many reviews and requested changes, the code turned into this and nobody noticed :stuck_out_tongue: 

Either way, I will take a closer look and fix this. Thanks for bringing this up :+1:

Reply:
Sorry for the delay. I've compared this method with the corresponding one in the `project` module, and indeed, the operator should be set to "not in" if the condition is not met. I've created a PR to master: https://github.com/odoo/odoo/pull/186167